test/specs/modules/Rating/Rating-test.js

```diff
@@ -1,190 +1,199 @@
 
       wrapper.should.have.className('selected')
     })
+
     it('selects icons up to and including the hovered icon', () => {
       const wrapper = mount(<Rating maxRating={3} />)
-      const icons = wrapper.find('.icon')
+      const icons = wrapper.find('RatingIcon')
 
       icons.at(1).simulate('mouseEnter')
 
-      icons.at(0).should.have.className('selected')
-      icons.at(1).should.have.className('selected')
-      icons.at(2).should.not.have.className('selected')
+      icons.at(0).should.have.prop('selected', true)
+      icons.at(1).should.have.prop('selected', true)
+      icons.at(2).should.have.prop('selected', false)
     })
+
     it('unselects icons on mouse leave', () => {
       const wrapper = mount(<Rating maxRating={3} />)
-      const icons = wrapper.find('.icon')
+      const icons = wrapper.find('RatingIcon')
 
       icons.last().simulate('mouseEnter')
       wrapper.simulate('mouseLeave')
 
-      icons.every('.icon.selected')
-        .should.equal(false, 'Some icon did not remove its "selected" class')
+      icons.findWhere(i => i.prop('selected', true))
+        .should.have.length(0)
     })
   })
 
   describe('clearable', () => {
     it('prevents clearing by default with multiple icons', () => {
       const icons = mount(<Rating defaultRating={5} maxRating={5} />)
-        .find('.icon')
+        .find('RatingIcon')
 
-      icons.find('.active').last().simulate('click')
+      icons.last().simulate('click')
 
-      icons.every('.icon.active')
-        .should.equal(true, 'Some icon did not retain its "active" class')
+      icons.findWhere((i) => i.prop('active', true))
+        .should.have.length(5)
     })
+
     it('allows toggling when set to "auto" with a single icon', () => {
       const icon = mount(<Rating maxRating={1} clearable='auto' />)
-        .find('.icon')
+        .find('RatingIcon')
         .at(0)
 
       icon
         .simulate('click')
-        .should.have.className('active')
+        .should.have.prop('active', true)
 
       icon
         .simulate('click')
-        .should.not.have.className('active')
+        .should.have.prop('active', false)
     })
+
     it('allows clearing when true with a single icon', () => {
       mount(<Rating defaultRating={1} maxRating={1} clearable />)
-        .find('.icon')
+        .find('RatingIcon')
         .at(0)
         .simulate('click')
-        .should.not.have.className('active')
+        .should.have.prop('active', false)
     })
+
     it('allows clearing when true with multiple icons', () => {
       const icons = mount(<Rating defaultRating={4} maxRating={5} clearable />)
-        .find('.icon')
+        .find('RatingIcon')
 
-      icons.find('.active').last().simulate('click')
+      icons.at(3).simulate('click')
 
-      icons.every('.icon.active')
-        .should.equal(false, 'Some icon did not remove its "active" class')
+      icons.findWhere((i) => i.prop('active', true))
+        .should.have.length(0)
```

Comment:
Looks like there are a lot of these so I'll skip commenting on the rest!  Though, let's update them all.  Note, the last argument to any assertion is the custom failure message.